packages/api-page-builder-aco/src/types.ts

```diff
@@ -0,0 +1,35 @@
+import { AcoContext } from "@webiny/api-aco/types";
+import { Page, PbPageElement } from "@webiny/api-page-builder/types";
+import { PbContext } from "@webiny/api-page-builder/graphql/types";
+import { Context as BaseContext } from "@webiny/handler/types";
+
+interface PageSearchProcessorParams {
+    page: Page;
+    block: PbPageElement;
+    element: PbPageElement;
+}
+
+export type PbPageRecordData = Pick<
+    Page,
+    | "id"
+    | "pid"
+    | "title"
+    | "createdOn"
+    | "createdBy"
+    | "savedOn"
+    | "status"
+    | "version"
+    | "locked"
+    | "path"
+>;
+
+export interface PageSearchProcessor {
+    (params: PageSearchProcessorParams): string;
+}
+
+export interface PbAcoContext extends BaseContext, AcoContext, PbContext {
+    pageBuilderAco: {
+        addPageSearchProcessor(processor: PageSearchProcessor): void;
+        processPageSearchContent(content: Page): Promise<string>;
```

Comment:
`processPageSearchContent` is a bit vague. Process how? See what I mean, what will happen after the processing takes place? It's unclear. Maybe call this `getSearchablePageContent`? 🤔

Reply:
Make sense 👍